src/utils/parse-path.ts

```diff
@@ -0,0 +1,25 @@
+export interface ParsedPath {
+  pathname: string
+  query: string
+  hash: string
+}
+
+export const parsePath = (path: string): ParsedPath => {
```

Comment:
@stephenjason89 You'd better to use this pattern to support `?`/`#` better in 
original `importPath` for https://github.com/import-js/eslint-plugin-import/pull/3177

Reply:
Got it.